extensions/amp-social-share/validator-amp-social-share.protoascii

```diff
@@ -14,8 +14,20 @@
 # limitations under the license.
 #
 
-tags: {  # amp-social-share
+tags: {  # amp-social-share in websites
   html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-social-share"
+    version: "0.1"
+    version: "1.0"
+    version: "latest"
+    requires_usage: EXEMPTED
```

Comment:
For new usage, either omit `requires_usage` (default is `ERROR`), or add `requires_usage: NONE` if the script tag has a useful function without any usage on the page.

EXEMPTED should really be called LEGACY_EXEMPTED (my fault), and is there because we don't want to invalidate existing AMP pages. These cases are handled by the [unused extensions transformer](https://github.com/ampproject/amppackager/blob/releases/transformer/transformers/unusedextensions.go).

Reply:
Discussed offline and changes made here: https://github.com/ampproject/amphtml/pull/33478/commits/d36f66f592880321b8e002180e30f1252d019c07